docs/f_fractal_dimension.md

```diff
@@ -0,0 +1,11 @@
+# Fractal dimension features
+
+Suppose $A$ is a shape's area and $P$ is its perimeter, and we are determining $D$.
+
+## FRACT_DIM_BOXCOUNT
+
+The underlying relation is $n = b^{-D}$. The feature is calculated as the slope $-D$ of the plot $\text {log} \: n$ against $\text {log} \: b$.

```

Comment:
It may be helpful to add what the variable b represents

Reply:
Details and math added